runtime/bcutil/cfreader.c

```diff
@@ -2442,8 +2460,7 @@ j9bcutil_readClassFileBytes(J9PortLibrary *portLib,
 		classfile->accessFlags |= CFR_ACC_ABSTRACT;
 	}
 
-	if ((vmVersionShifted >= BCT_Java9MajorVersionShifted)
-			&& (J9_ARE_ALL_BITS_SET(classfile->accessFlags, CFR_ACC_MODULE))) {
+	if ((classfile->majorVersion >= 53) && (classfile->accessFlags == CFR_ACC_MODULE)) {
```

Comment:
Why the change to use classfile->majorVersion instead of vmVersionShifted? If its required, please add comments to explain it, and create or use an existing defined constant for 53. 

I still don't think its correct to check `accessFlags == CFR_ACC_MODULE` rather than checking if the CFR_ACC_MODULE bit is set.

Reply:
Tests were failing in some cases using vmVersionShifted because a class file with version below 53 can be loaded by a Java 9 vm and this check should not apply.